Content.Server/GameTicking/Rules/GameRuleSystem.cs

```diff
@@ -37,25 +38,31 @@ private void OnGameRuleAdded(GameRuleAddedEvent ev)
             return;
 
         Enabled = true;
-        Added();
     }
 
-    private void OnGameRuleRemoved(GameRuleRemovedEvent ev)
+    private void OnGameRuleStarted(GameRuleStartedEvent ev)
     {
         if (ev.Rule.ID != Prototype)
             return;
 
-        Enabled = false;
-        Removed();
+        Started();
+    }
+
+    private void OnGameRuleEnded(GameRuleEndedEvent ev)
+    {
+        if (ev.Rule.ID != Prototype)
+            return;
+
+        Ended();
```

Comment:
missing a `Enabled = false;`, leading to infinite restart-round loops.
Probably the cause of the weird test error